compiler/luci/import/src/Importer.test.cpp

```diff
@@ -16,13 +16,289 @@
 
 #include "luci/Importer.h"
 
-#include <loco.h>
+#include <luci/IR/CircleNode.h>
+#include <luci/Plan/CircleNodeExecutionPlan.h>
 
 #include <gtest/gtest.h>
+#include <mio/circle/schema_generated.h>
+#include <flatbuffers/flatbuffers.h>
 
 TEST(TensorFlowLiteImport, Dummy)
 {
   luci::Importer import;
 
   SUCCEED();
 }
+
+// helpers for flatbuffers
+namespace
+{
+
+struct BasicCircleModel
+{
+  std::unique_ptr<circle::ModelT> model;
+
+  BasicCircleModel()
+  {
+    model = std::make_unique<circle::ModelT>();
+    model->buffers.emplace_back(new circle::BufferT);
```

Comment:
I investigated this a little and found interesting thing.

`model->buffers.emplace_back(new circle::BufferT);` is generally unsafe, it can leak memory:
if relocation for larger buffer fails, exception is thrown. In this case memory allocated by `new` is lost.

Therefore your fears are reasonable!

It is unlikely case, but still interesting  =)